### ISSUE TITLE
[Bug]: ButtonGroup type mutiple data-state does't get updated 

### ISSUE THREAD

**Issue by LukaPelgrom** [radix-vue/radix-vue#768]
### Environment

```bash
Developement/Production OS: Macbook pro m1
Node version: 16.0.0
Package manager: pnpm@8.6.0
Radix Vue version: 1.5.2
Vue version: latest
CSS framework: latest
Browser: Chrome 90.0.4430.212
```

### Link to minimal reproduction

https://stackblitz.com/edit/uhbmps?file=src%2FApp.vue

### Steps to reproduce

- [ ] Open stackblitz
- [ ] toggle the button

### Describe the bug

The data-state toggle is not working in my project and in the stackblitz which is provided on the website https://www.radix-vue.com/components/toggle-group.html#root.

the state binded with v-model isn't updating either (like the single), but the @update:modelValue="(v) => console.log(v)" is providing the state. 

### Expected behavior

I expect the data-state toggle in sync with data-active as in the example on the website. I downgraded to version 1.4.9 and it works like expected.

<img width="1374" alt="image" src="https://github.com/radix-vue/radix-vue/assets/125878661/a725c389-eedc-456a-9024-02c2fe45827c">

### Context & Screenshots (if applicable)

<img width="1446" alt="image" src="https://github.com/radix-vue/radix-vue/assets/125878661/5035a495-ba95-40af-9faa-27f9fc899104">

### Related 
https://github.com/radix-vue/radix-vue/issues/754

**Comment by zernonia** [comment]
Duplicate of https://github.com/radix-vue/radix-vue/issues/754. Will publish the fixed version soon

**Comment by LukaPelgrom** [comment]
Hallo, I see you have fixed it in version 1.5.3 thank you!!
